modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -92,6 +92,11 @@
 public class SqlQueryProcessor implements QueryProcessor {
     private static final IgniteLogger LOG = Loggers.forClass(SqlQueryProcessor.class);
 
+    /**
+     * Default planner timeout, in ms.
+     */
+    private static final long PLANNER_TIMEOUT = 15000L;
```

Comment:
```suggestion
    /** Default planner timeout, in ms. */
    private static final long PLANNER_TIMEOUT = 15000L;
```
Do we plan to make it configurable?

Reply:
yes, we have a separate ticket for that purpose